docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -300,18 +300,18 @@ kubectl create secret generic <secret-name> \
 kubectl apply -f ./hot-backup-agent.yaml
 ----
 
-=== Scheduling External Backups
+=== Scheduling External Backups with CronHotBackup Custom Resource
 
-You can schedule external backups, using the `spec.schedule` field of a `HotBackup` resource.
+You can schedule external backups, using the `.spec.schedule` and `.spec.hotBackupTemplate` fields of the `CronHotBackup` resource. You can find more detailed information about the `CronHotBackup` resource in the xref:api-ref.adoc#cronhotbackupspec[API Reference] page.
 
-. Create the `HotBackup` resource with a `schedule` field:
+. Create the `CronHotBackup` resource with `schedule` and `hotBackupTemplate` fields.
 +
 [source,yaml,subs="attributes+"]
 ----
-include::ROOT:example$/hot-backup-agent-scheduled.yaml[]
+include::ROOT:example$/cron-hot-backup-agent.yaml[]
 ----
 +
-This field takes a valid cron expression. For example:
+`.spec.schedule` field takes a valid cron expression. For example:
 +
```

Comment:
For the table below, just some suggestions for slight rewordings.

At 10.30 AM every day

On 1st, 15th, and 25th of each month at midnight

On the first day of each month at midnight

For a full list of support expressions, see the library documentation.

Reply:
I am updating it